stack.yaml

```diff
@@ -1,4 +1,10 @@
-resolver: lts-18.2
+resolver: lts-18.18
+
+extra-deps:
+  - hspec-2.9.3
+  - hspec-core-2.9.3
+  - hspec-discover-2.9.3
+  - hspec-junit-formatter-1.1.0.0
```

Comment:
Does this be `hspec-junit-formatter-1.0.3.0` then?

Reply:
I tested it with that, but decided to still use latest for local development.

Reply:
Similarly, we work fine with hspec-2.8, but I added 2.9 here anyway.